community/create-a-release.md

```diff
@@ -280,6 +280,14 @@ docker buildx inspect --bootstrap
 docker buildx build --platform linux/amd64,linux/arm64 --tag ${your_dockerhub_username}/kvrocks:${release_version} --tag ${your_dockerhub_username}/kvrocks:latest . --output "type=registry" --build-arg MORE_BUILD_ARGS=-j12 --build-arg http_proxy=${your_http_proxy} --build-arg https_proxy=${your_https_proxy}
 ```
 
+## Draft release note
```

Comment:
Try to add the draft link in the voting email template.

Reply:
This is already included:

```markdown
Git tag for the release:
https://github.com/apache/kvrocks/releases/tag/${release_version}-rc${candidate_number}
```

Reply:
Interesting, previously we don't even instruct to draft a release but actually the template contains it ..